src/shogun/multiclass/KNNSolver.h

```diff
@@ -16,9 +16,7 @@
 namespace shogun
 {
 /**
- * The KNNSolver class is the virtual base class of BruteKNNSolcer, CoverTreeKNNSolver, KDTreeKNNsolver, LSHKNNSolver. 
- * The KNNSolver class will include most of Variables which been used all the other Solver and methods been used to choose 
- * the index of the most frequent class.
+ * The KNNSolver class is the virtual base class of all the other solvers.
```

Comment:
"Virtual base class for all KNN solvers"

No need to put the class name in the description of a class, the name is obvious from the API